sfaira/train/summaries.py

```diff
@@ -1162,19 +1176,20 @@ def __init__(
 
     def create_summary_tab(self):
         metrics = list(self.evals.values())[0]['val'].keys()
+        run_ids = [run_id.replace('-', '_') for run_id in self.run_ids]
         self.summary_tab = pandas.DataFrame(dict(
             list({
-                "depth": [id_i.split("_")[self.model_id_len + 0] for id_i in self.run_ids],
-                "width": [id_i.split("_")[self.model_id_len + 1] for id_i in self.run_ids],
-                "lr": [id_i.split("_")[self.model_id_len + 2] for id_i in self.run_ids],
-                "dropout": [id_i.split("_")[self.model_id_len + 3] for id_i in self.run_ids],
-                "l1": [id_i.split("_")[self.model_id_len + 4] for id_i in self.run_ids],
-                "l2": [id_i.split("_")[self.model_id_len + 5] for id_i in self.run_ids],
-                "cv": [id_i.split("_")[-1] if self.cv else "1" for id_i in self.run_ids],
-                "model": ["_".join(id_i.split("_")[:self.model_id_len]) for id_i in self.run_ids],
-                "organ": [id_i.split("_")[2] for id_i in self.run_ids],
-                "model_type": [id_i.split("_")[3] for id_i in self.run_ids],
-                "model_gs_id": ["_".join(id_i.split("_")[:(self.model_id_len + 6)]) for id_i in self.run_ids],
+                "depth": [id_i.split("_")[self.model_id_len + 0] for id_i in run_ids],
+                "width": [id_i.split("_")[self.model_id_len + 1] for id_i in run_ids],
+                "lr": [id_i.split("_")[self.model_id_len + 2] for id_i in run_ids],
+                "dropout": [id_i.split("_")[self.model_id_len + 3] for id_i in run_ids],
+                "l1": [id_i.split("_")[self.model_id_len + 4] for id_i in run_ids],
+                "l2": [id_i.split("_")[self.model_id_len + 5] for id_i in run_ids],
+                "cv": [id_i.split("_")[-1] if self.cv else "1" for id_i in run_ids],
+                "model": ["_".join(id_i.split("_")[:self.model_id_len]) for id_i in run_ids],
+                "organ": [id_i.split("_")[2] for id_i in run_ids],
+                "model_type": [id_i.split("_")[3] for id_i in run_ids],
+                "model_gs_id": self.run_ids,
```

Comment:
"model_gs_id" and "run" now have the same value, not sure how much sense that makes?